cves/CVE-2011-1184.yml

```diff
@@ -43,41 +43,47 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
```

Comment:
Was there sensitive information being sent between the client and server? Or were these messages only used to obtain authentication?